__tests__/app.test.js

```diff
@@ -41,3 +41,40 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 responds with an article object, which contains all the necessary properties", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then((response) => {
+        const article = response.body;
+        expect(typeof article).toBe("object");
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.article_id).toBe("number");
```

Comment:
It would be beneficial here to assert exactly what number you expect the article id to be, as currently this test will pass even if it returns the wrong article